charts/bandstand-cron-job/templates/_helpers.tpl

```diff
@@ -24,8 +24,8 @@ capabilities:
 {{- end -}}
 
 {{- define "bandstand-cron-job.podSecurityContext" -}}
-runAsUser: 1000
-fsGroup: 1000
+runAsUser: {{ .Values.runAsUser | default 1000  }}
+fsGroup: {{ .Values.fsGroup | default 1000  }}
```

Comment:
docs? or is this an undocumented feature just for us?

Reply:
Mostly it should not be necessary I would hope